site/docs/components/progress/usage.mdx

```diff
@@ -9,35 +9,34 @@ data:
   $ref: ./#/data
 ---
 
-### Using the Progress component
+### Using the `CircularProgress` and `LinearProgress` component
 
-It should be used when the operation will take more than a second to complete, to reassure users that the operation is in progress.
+#### When to use `CircularProgress` and `LinearProgress`
 
-#### When not to use Progress
+- When it’s possible to determine the length of time remaining for a task or operation to complete. This will reassure the user that it is being processed.
+- To provide users with a visual indication of the status of an operation, or how much of a process is completed.
 
-Do not use when an operation takes a short amount of time, as it may be confusing and distracting for users.
+#### When not to use `CircularProgress` and `LinearProgress`
 
-### Import
+- When a task or operation will take an indeterminate length of time to complete. Instead, use [`Spinner`](/salt/components/spinner).
+- To indicate the loading of content on navigation. Instead, use [`Spinner`](/salt/components/spinner).
 
-{/* Update the text and code snippet below as needed: */}
+### Import
 
-To import the Progress components from the core Salt package, use:
+To import the `CircularProgress` and `LinearProgress` components from the lab Salt package, use:
 
 ```js
-import { CircularProgress } from "@salt-ds/core";
+import { CircularProgress } from "@salt-ds/lab";
 // or
-import { LinearProgress } from "@salt-ds/core";
+import { LinearProgress } from "@salt-ds/lab";
 ```
 
 ### Props
 
-{/* Update packageName and componentName below as needed */}
-{/* packageName is optional and defaults to "core" if omitted */}
-
-#### Circular Progress
+#### `CircularProgress`
 
-<PropsTable packageName="core" componentName="CircularProgress" />
+<PropsTable packageName="lab" componentName="CircularProgress" />
 
-#### Linear Progress
+#### `LinearProgress`
 
-<PropsTable packageName="core" componentName="LinearProgress" />
+<PropsTable packageName="lab" componentName="LinearProgress" />
```

Comment:
`unit` prop description needs a bit more info. Add or remove `aria-label` to be consistent with circular.